.gitignore

```diff
@@ -23,3 +23,6 @@ surface_src/java_src/Alice/src/com/roboclub/robobuggy/main/config.java
 real_time/arduino_src/radio_buggy_mega/fingerprint.h
 
 real_time/surface_src/java_src/Alice/.classpath
+
+#eclipse's nonsense
```

Comment:
Add a space here, so it's like the rest of the comments?

# eclipse generated files
real_time/surface_src/java_src/Alice/.classpath
real_time/surface_src/java_src/.recommenders/
**/.metadata/*

That last one should keep all .metadata out, no matter where they end up in the tree. Maybe the same should go for .recommenders, if that's something eclipse decides where to put based on workspace?

Add a comment about fingerprint for good hygiene?